metadata-ingestion/tests/unit/test_bigquery_source.py

```diff
@@ -93,11 +95,11 @@ def test_simple_upstream_table_generation():
         }
     )
     source = BigqueryV2Source(config=config, ctx=PipelineContext(run_id="test"))
-    lineage_metadata = {str(a): {str(b)}}
+    lineage_metadata = {str(a): {LineageEdge(table=str(b), created=datetime.now())}}
     upstreams = source.lineage_extractor.get_upstream_tables(
         str(a), lineage_metadata, []
     )
-    assert list(upstreams) == [b]
+    assert list(upstreams)[0].table == str(b)
```

Comment:
Maybe also assert the list length is 1 since we're losing that check now. Or we could set the `LineageEdge` to a variable and check `list(upstreams) == [edge]` right? Same goes for all tests, although I guess gets more complicated with sorting. Just thinking out loud, no changes needed here